src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+  &__list {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__element:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    padding: 23px 0;
+    float: left;
+    position: relative;
+    text-decoration: none;
+    color: $color_dark_violet;
+    font-weight: 500;
+    font-size: 12px;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $color_blue;
+    }
+
+    &::after {
+      content: "";
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      width: 0;
+      height: 4px;
+      background: $color_blue;
+      border-radius: 8px;
+    }
+
+    &:focus::after,
+    &.is-active::after {
+      width: 100%;
+    }
+
+    &.is-active {
+      color: $color_blue;
+    }
```

Comment:
Implement it only with is-active class
![image](https://user-images.githubusercontent.com/122989585/215741434-b5e530e4-1657-42bc-9d65-ff29803a2bba.png)